test/verify/check-networkmanager-settings

```diff
@@ -186,6 +186,44 @@ class TestNetworkingSettings(NetworkCase):
         b.click('#network-ip-settings-cancel')
         b.wait_not_present("#network-ip-settings-dialog")
 
+    def testIpHelper(self):
+        b = self.browser
+
+        self.login_and_go("/network")
+        b.wait_visible("#networking")
+
+        iface1 = "cockpit1"
+        self.add_veth(iface1)
+
+        b.click("button:contains('cockpit1')")
+        b.wait_in_text("#network-interface-name", "cockpit1")
+
+        # wait until dialog initialized
+        b.click("#networking-edit-ipv4:contains('edit')")
+        b.wait_visible("#network-ip-settings-dialog")
+
+        b.select_from_dropdown("#network-ip-settings-select-method", "manual")
+        b.wait_visible("#network-ip-settings-address-0")
+        # test class A
+        b.set_input_text("#network-ip-settings-address-0", "10.0.5.1")
+        b.wait_val("#network-ip-settings-netmask-0", "255.0.0.0")
+        b.wait_val("#network-ip-settings-gateway-0", "10.0.5.254")
+        # test class B
+        b.set_input_text("#network-ip-settings-address-0", "172.16.44.2")
+        b.wait_val("#network-ip-settings-netmask-0", "255.255.0.0")
+        b.wait_val("#network-ip-settings-gateway-0", "172.16.44.1")
+        # test class C
+        b.set_input_text("#network-ip-settings-address-0", "192.168.1.1")
+        b.wait_val("#network-ip-settings-netmask-0", "255.255.255.0")
+        b.wait_val("#network-ip-settings-gateway-0", "192.168.1.254")
+        # free manual edit
+        b.set_input_text("#network-ip-settings-netmask-0", "255.255.255.128")
+        b.set_input_text("#network-ip-settings-gateway-0", "192.168.1.126")
+
+        b.wait_val("#network-ip-settings-address-0", "192.168.1.1")
+        b.wait_val("#network-ip-settings-netmask-0", "255.255.255.128")
+        b.wait_val("#network-ip-settings-gateway-0", "192.168.1.126")
+
```

Comment:
It's a bit unclear what this does -- as such it just checks that the values are exactly the ones that you just set. So this doesn't effectively test anything. I suppose you want to ensure that it does not change the values automatically once an input line got a customized value, but that is a race condition.

I suggest to actually confirm/close the dialog, and validate the setting with either `nmcli` or inspecting the properties in the UI, as you prefer. Then you know that they *really* got applied.

Reply:
ack.
I needed to activate the interface with dhcp_cidr parameter defined `self.add_veth(iface, dhcp_cidr="10.111.113.2/20")` because if `dhcp_cidr` is not set interface isn't activated.
Maybe this line can go out of _if scope_, but I don't know if it has more involvement
https://github.com/cockpit-project/cockpit/blob/8898cefcf66fa05c588cd6b760917f42f88bfc6a/test/verify/netlib.py#L44